aries_vcx/src/handlers/out_of_band/sender/sender.rs

```diff
@@ -54,33 +54,18 @@ impl OutOfBandSender {
 
     pub fn append_a2a_message(mut self, msg: A2AMessage) -> VcxResult<Self> {
         let (attach_id, attach) = match msg {
-            A2AMessage::CredentialRequest(request) => {
```

Comment:
Removed support for some of these messages, the only expected OOB messages are going to contain `credential offer` and `presentation request`.